scripts/datascience/themes_quality/notebooks/run_full_investigation.ipynb

```
@@ -0,0 +1,692 @@
+    "        timeout=600  # 10 minute timeout\n",
+    "    )\n",
+    "    \n",
+    "    print(\"\ud83d\udccb Parameter Sweep Output:\")\n",
+    "    print(result.stdout)\n",
+    "    \n",
+    "    if result.stderr:\n",
+    "        print(\"\\n\u26a0\ufe0f Warnings/Errors:\")\n",
+    "        print(result.stderr)\n",
+    "    \n",
+    "    if result.returncode == 0:\n",
+    "        print(\"\\n\u2705 Parameter sweep completed successfully\")\n",
```

Ruff (0.12.2)

Undefined name `display`

(F821)